internal/tools/helm/run.go

```diff
@@ -5,20 +5,37 @@
 package helm
 
 import (
+	"os"
 	"os/exec"
 	"strings"
 
 	"github.com/mattermost/mattermost-cloud/internal/tools/exechelper"
 	log "github.com/sirupsen/logrus"
 )
 
+const helmLoggingEnvironmentVariable string = "MM_CLOUD_VERBOSE_HELM_OUTPUT"
+
+// SetVerboseHelmLogging controls an environment variable which
+// signals whether or not the stdout output from Helm should be
+// included as DEBUG level logs or not. True turns them on, false
+// turns them off.
+func SetVerboseHelmLogging(enable bool) {
+	if enable {
+		os.Setenv(helmLoggingEnvironmentVariable, "true")
+	} else {
+		os.Setenv(helmLoggingEnvironmentVariable, "")
+	}
+}
+
 func outputLogger(line string, logger log.FieldLogger) {
 	line = strings.TrimSpace(line)
 	if len(line) == 0 {
 		return
 	}
 
-	logger.Debugf("[helm] %s", line)
+	if os.Getenv("MM_CLOUD_VERBOSE_HELM_OUTPUT") != "" {
+		logger.Debugf("[helm] %s", line)
+	}
```

Comment:
I would like to see if we can change the implementation of this if possible. Although I see how this would work, we are doing a bunch of logic checks in this function for each log line generated when it seems like we just wouldn't want to call this at all.

I believe we should be able to build a helm logger that simply skips calling this function entirely if we aren't logging. As part of this, I think it should be possible to avoid setting a global env var that requires the `expandedArgs = append(expandedArgs, os.ExpandEnv(arg))` logic above.

Reply:
Uh @gabrieljackson the call to `os.ExpandEnv` is in a different hunk. Did you comment on the wrong one?

Reply:
Resolved in dc3817c